backend/tests/conftest.py

```
@@ -0,0 +1,27 @@
+    """Очистка коллекции MongoDB перед каждым тестом"""
+    await db_manager.connect()
+    await db_manager.collection.delete_many({})
+    
+    # Очистка папки uploads
+    upload_dir = os.getenv("DATASET_PATH", "uploads")
+    if os.path.exists(upload_dir):
+        for f in os.listdir(upload_dir):
+            try:
+                os.remove(os.path.join(upload_dir, f))
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)